src/mprk.jl

```diff
@@ -208,7 +168,7 @@ function alg_cache(alg::MPE, u, rate_prototype, ::Type{uEltypeNoUnits},
     if !(f isa PDSFunction || f isa ConservativePDSFunction)
         throw(ArgumentError("MPE can only be applied to production-destruction systems"))
     end
-    MPEConstantCache(floatmin(uEltypeNoUnits))
+    MPEConstantCache(convert(uEltypeNoUnits, alg.small_constant))
```

Comment:
Continuing the suggestion above, this would become

```suggestion
    MPEConstantCache(alg.small_constant(uEltypeNoUnits))
```

If you want to set a fixed value manually, you could do something like
```julia
MPE(small_constant = Returns(1.0e-50))
```

If you like this approach, we should also adapt the docstring accordingly.